index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/dcd34e45ea.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="styles.css">
+    <title>Global Summit</title>
+</head>
+<body>
+
+        <header>
+            <div class="upper-links">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <a href="#">Logout</a>
+                <a href="#">Mypage</a>
+                <a href="#">Korean</a>
+            </div>
+            <nav class="nav-bar">
+                    <div id="logo"><img src="./images/summit-icon.jpg"></div>
+                        <ul>
+                            <li><a href="./about.html">About</a></li>
+                            <li><a href="#program">Program</a></li>
+                            <li><a href="#join">Join</a></li>
+                            <li><a href="#sponsor">Sponsor</a></li>
+                            <li><a href="#news">News</a></li>
+                            <li><a href="#campaign">CC Campaign</a></li>
+                        </ul>
+                    <div class="accordion">
+                            <div class="bar1"></div>
+                            <div class="bar2"></div>
+                            <div class="bar3"></div>
+                    </div>
+            </nav>
+
+            <div id="panel">
+                <a class="drop-link" href="./about.html">About</a>
+                <a class="drop-link" href="#program">Program</a>
+                <a class="drop-link" href="#join">Join</a>
+                <a class="drop-link" href="#sponsor">Sponsor</a>
+                <a class="drop-link" href="#news">News</a>
+                <a class="drop-link" href="#campaign">CC Campaign</a>
+            </div>
```

Comment:
- [ ] I think it would be better to use an unordered list here too for semantic HTML and SEO rules :recycle:

Reply:
Converted them to unordered list as instructed